src/test/java/uk/gov/hmcts/reform/roleassignment/domain/service/drools/DroolJudicialCategoryTest.java

```diff
@@ -108,6 +109,7 @@ void shouldRejectOrgValidation_MissingAttributeJurisdiction() {
             roleAssignment.setRoleType(RoleType.ORGANISATION);
             roleAssignment.setRoleName("judge");
             roleAssignment.setGrantType(STANDARD);
+            roleAssignment.setStatus(Status.CREATE_REQUESTED);
```

Comment:
same here..

Reply:
This was added so that the request will meet the drool rule condition as the test case it's trying to achieve